_posts/2021-03-06-newpipe-0.20.11-released.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "NewPipe 0.20.11 released: Hotfixing YouTube"
+short: "NewPipe 0.20.10 released"
+date: 2021-03-06 11:00:00 + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Welcome to the first hotfix release of 2021 - let us hope it will be the last one. This release fixes receiving some YouTube content, an app crash and brings you a new translation.
+
+<!-- more -->
+
+## Changes
+
+A short release, so this blog post is short as well. These are the fixes which found their way into this release:
+
+* [YouTube] getting related videos from channels ([@TiA4f8R](https://github.com/TiA4f8R)) and playlists [@XiangRongLin](https://github.com/XiangRongLin)
+* [YouTube] getting comments [@golfinq](https://github.com/golfinq)
+* [SoundCloud] getting geo restricted content as well as the client id [@TobiGr](https://github.com/TobiGr)
+* [App] crash when reordering channel groups [@Isira-Seneviratne](https://github.com/Isira-Seneviratne)
```

Comment:
Why is it changelog style? If you don't write sentences (that can explain technical thing, why, how, that are more readable than changelog, whatever you want…), I don't see the benefit to read a blog post rather than reading changelog.txt